### ISSUE TITLE
static-site: All women are now "GivenName SurName (nÃ©e MaidenName)"

### ISSUE THREAD

**Issue by jhannah** [nigelhorne/ged2site#123]
Hmm, something has gone sideways in `master` 01a2602796c30ce7ca021f0b89c937722d345c0b. e.g.: http://jays.net/genealogy/static-site/I2310.html

**Comment by nigelhorne** [comment]
That's surprising, given that particular commit backed out of the change which could have caused this problem.  Can you confirm it was that commit, not one just before it, which caused that?  I am unable to reproduce.

**Comment by jhannah** [comment]
Huh. For whatever reason, I'm having to add this to get multiple browsers to render it correctly:

```
<html>
<head>
  <meta charset="utf-8">
```

**Comment by nigelhorne** [comment]
That shouldn't be needed, since the code should emit "&eacute;" for that character.

**Comment by nigelhorne** [comment]
Are you using a fork or my master?  If a fork, please check that the code in wide_to_html() is up-to-date with the master.  If it's the master, can I borrow a brick wall to hit my head against?  This just doesn't make sense.

**Comment by jhannah** [comment]
Ya, apologies, I was completely mystified by this. When I ran `cat` on macOS or Debian Linux, I saw `Sharon Erlene Hannah (née Humphrey)`. It looked fine. Using `vim` on macOS and Debian it looked fine. Then I'd load it in my web browser (3 different devices) and get the crazy characters (locally on macOS, and after moving the files to Debian Linux). I have lots of brick walls here, I was using them. 🙂 I shoved `<meta charset="utf-8">` into the .html files, and the problem went away. (branch `112-and-115` in my fork.) I can try again off your new master the next time I run it and see if my hacks can be thrown away.
